docs/developer/database-schema-migrations.md

```diff
@@ -29,10 +29,16 @@ ADD COLUMN `id` int(11) NOT NULL AUTO_INCREMENT FIRST,
 DROP PRIMARY KEY,
 ADD PRIMARY KEY (`id`) USING BTREE,
 ADD UNIQUE INDEX(`varname`);
-)"
+)",
+		.content_schema_update = false,
+		.force_interactive = true
 	},
 ```
 
+### Optional fields:
+- `.content_schema_update`	- If true, this will update the content_db if configured
```

Comment:
We should leave this field in all the time, otherwise it often gets forgotten in PR's